samples/noise3D.js

```diff
@@ -0,0 +1,316 @@
+(function(exports) {
+    "use strict";
+    /*global window,vec3*/
+
+    exports = exports || window;
+
+    function step(edge, x) {
+        return [
+            (x[0] < edge[0]) ? 0.0 : 1.0,
+            (x[1] < edge[1]) ? 0.0 : 1.0,
+            (x[2] < edge[2]) ? 0.0 : 1.0
+        ];
+    }
+
+    function step_vec4(edge, x) {
+        return [
+            (x[0] < edge[0]) ? 0.0 : 1.0,
+            (x[1] < edge[1]) ? 0.0 : 1.0,
+            (x[2] < edge[2]) ? 0.0 : 1.0,
+            (x[3] < edge[3]) ? 0.0 : 1.0
+        ];
+    }
+
+    function min(x, y) {
+        return [
+            y[0] < x[0] ? y[0] : x[0],
+            y[1] < x[1] ? y[1] : x[1],
+            y[2] < x[2] ? y[2] : x[2]
+        ];
+    }
+
+    function max(x, y) {
+        return [
+            y[0] > x[0] ? y[0] : x[0],
+            y[1] > x[1] ? y[1] : x[1],
+            y[2] > x[2] ? y[2] : x[2]
+        ];
+    }
+
+    function max_vec4(x, y) {
+        return [
+            y[0] > x[0] ? y[0] : x[0],
+            y[1] > x[1] ? y[1] : x[1],
+            y[2] > x[2] ? y[2] : x[2],
+            y[3] > x[3] ? y[3] : x[3]
+        ];
+    }
+
+    function vec4_dot(left, right) {
+        return left[0] * right[0] +
+        left[1] * right[1] +
+        left[2] * right[2] +
+        left[3] * right[3];
+    }
+
+    //
+    // Description : Array and textureless GLSL 2D/3D/4D simplex
```

Comment:
Can you please move this to LICENSE.md (and format it similar to the other third-party libraries)?